docs.md

```diff
@@ -9,32 +12,59 @@ An API providing data for the four apps on the Nazareth Open Tourism Platform, b
 - [Update event](#update-event)
 - [Delete event](#delete-event)
 
+**Places**
+- [Get all places](#get-all-places)
+- [Get place by id](#get-place-by-id)
+- [Create place](#create-place)
+- [Update place](#update-place)
+- [Delete place](#delete-place)
+
+**Products**
+- [Get all products](#get-all-products)
+- [Get product by id](#get-product-by-id)
+- [Create product](#create-product)
+- [Update product](#update-product)
+- [Delete product](#delete-product)
+
+**Users**
+- [Get all users](#get-all-users)
+- [Get user by id](#get-user-by-id)
+- [Create user](#create-user)
+- [Update user](#update-user)
+- [Delete user](#delete-user)
+
 ## Events
 
 ### Get all events
```

Comment:
Okay, so here we are missing the query params on the get requests.

Ie `GET /events?locationId={}`